ansible_community_steering_committee.rst

```diff
@@ -45,50 +42,36 @@ Members:
 10. Dylan Silva (``thaumos``)
 11. Brian Scholer (``briantist``)
 
--  Currently, all the Steering Committee members are selected based on
-   their active contribution to the Ansible Project.
--  In future, the selection process will be based on the election model
-   through community.
+Committee members are selected based on their active contribution to the Ansible Project and its Community.
 
 See the `Steering Committee Membership Guidelines <https://github.com/ansible/community-docs/blob/main/steering_committee_membership_guidelines.rst>`_ to learn details.
 
 New Policy/Proposals & Inclusion Requests:
 ==========================================
 
-Requests regarding new collection inclusion requests should be submitted
-via this `GitHub discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
+* The Committee uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.
+
+* If you want to discuss an idea that can impact the community, collection best practices, inclusion policy, submitted new policy/proposals & new collection inclusion requests or other topics the Committee is responsible for, create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ as a discussion topic, and it’ll be discussed asynchronously in the issue itself.
```

Comment:
I am struggling to come up with a proposed edit but I wonder if the list of things in this sentence is long enough to consider making it a bulleted list instead? Or maybe re-wording a little to make the list read easier.

Reply:
Maybe something like :
Create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ as a discussion topic if you want to discuss an idea that impacts any of the following:
 - Ansible community
 - collection best practices
 - blablabla

Reply:
Agreed, thanks! Changed PTAL